specification/databox/resource-manager/readme.ruby.md

```diff
@@ -18,6 +18,7 @@ batch:
   - tag: package-2020-11
   - tag: package-2021-03
   - tag: package-2021-05
+  - tag: package-2021-08-preview
```

Comment:
i don't think we will release new packages for ruby anymore.
Do you have this SDK release requirement?